haas_websocket/rest/haas_rest_handler.py

```diff
@@ -3,7 +3,7 @@
 import requests
 import logging
 from dotenv import load_dotenv
-from google.cloud import secretmanager
+from google.cloud import secretmanager_v1
```

Comment:
Is the v1 going to be deprecated soon or are we good to switch to using that?

Reply:
I changed my versions of secret manager to the version that the RTDH uses, and the import changed accordingly. I am hard coding the version number in my requirements.txt so this import shouldn't change anymore.

Reply:
Ah ok, that makes sense then.

> > In main.py, we're hard-coding to logging level INFO. Should this be configurable from an environment variable and just default back to INFO if none found? We do this in the other CV projects already for reference.
> 
> Are you talking about line 81 in main.py? I thought that this only pushed logging messages to the standard out (console) that were of the level info or more severe.

You are correct, this will log anything at or above INFO level. The standard out is picked up by the Google systems and logged to their own logging environment which they naturally charge for. In some of the deployments with the connector applications talking to our Kafka system we found INFO to be too robust for various environments and have allowed a config level change on this so we can tweak it on the fly. Since this is going to be deployed within the RTDH environment, I thought that would be a good approach here as well.